datasets/gov/lsd/statistika/bendrasis_gimstamumo_rodiklis.csv

```diff
@@ -0,0 +1,15 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
```

Comment:
ar įrašei .csv failo path'ą į get_data_gov_lt.in'ą?

Reply:
Sutvarkiau